pom.xml

```diff
@@ -37,16 +37,16 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <gml-v_3_1_1-schema.version>1.1.0</gml-v_3_1_1-schema.version>
```

Comment:
Are these used anymore?  These are just the jvnet pgc schema right?  These variables seem confusing